security/cmd/flexvolume/flexvolume.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
We have this one already, https://github.com/istio/istio/blob/master/security/docker/start_driver.sh